core/src/main/scala/kafka/server/DynamicConfig.scala

```diff
@@ -36,30 +35,12 @@ import scala.jdk.CollectionConverters._
 object DynamicConfig {
```

Comment:
It seems `DynamicConfig` gets smaller now. Maybe we can move it to server module in next PR. WDYT?

Reply:
It need to be moved anyway when we move `KafkaConfig` class. So it's on my roadmap for 
KAFKA-15853. The Jira for moving KafkaConfig out of core is one of the most complicated one as many pieces will need to move to support this.